app/views/diary_entries/show.html.erb

```diff
@@ -1,3 +1,5 @@
+ <%= javascript_include_tag "social_share_button" %>
```

Comment:
```suggestion
<% content_for :head do %>
  <%= javascript_include_tag "social_share_button" %>
<% end %>
```

That's how we do it, for example, in https://github.com/openstreetmap/openstreetmap-website/blob/fe81ac334cfef04751a959297dca24e472cb025f/app/views/dashboards/show.html.erb#L13-L15

Reply:
Thanks for the tip. Updated now.

Reply:
But it's not updated.

Reply:
Updated now, forgot to push, sorry.